src/commands/synthetics/utils.ts

```diff
@@ -875,21 +938,27 @@ export const renderResults = ({
     }
   }
 
-  const sortedResults = results.sort(sortResultsByOutcome())
+  for (const result of results) {
+    if (result.executionRule !== ExecutionRule.SKIPPED) {
+      summary.expected++
+    }
 
-  for (const result of sortedResults) {
     if (!config.failOnTimeout && result.timedOut) {
       summary.timedOut++
     }
 
-    if (result.result.unhealthy && !config.failOnCriticalErrors) {
+    if (hasResult(result) && result.result.unhealthy && !config.failOnCriticalErrors) {
       summary.criticalErrors++
     }
 
     const resultOutcome = getResultOutcome(result)
 
     if ([ResultOutcome.Passed, ResultOutcome.PassedNonBlocking].includes(resultOutcome)) {
       summary.passed++
+    } else if (resultOutcome === ResultOutcome.PreviouslyPassed) {
+      summary.expected++
```

Comment:
In this special case of "skipped" result, `datadog-ci` was expecting a result **to be run** but it was not run: it was found in a previous batch. We use this new `expected` property in the Continuous Testing Summary:

https://github.com/DataDog/datadog-ci/blob/1d8e7821d132076a3237dc86f3bb185a355f3814/src/commands/synthetics/reporters/default.ts#L380-L384